app/views/posts/index.html.erb

```diff
@@ -1,3 +1,47 @@
-<% 5.times do%>
-  <h2>Post index view</h2>
-<% end %>
\ No newline at end of file
+<main>
+  <section class ="post-section">
+    <h2>Post information for user</h2>
+    <article class="user-info">
+      <div class="user-card">
+        <div class="img-container">
+          <img src="<%=@user.photo || image_url('user_icon.png') %>" alt="Image of user <%=@user.name %>" title="<%=@user.name %>" class="user-image">
+        </div>
+        <div class="name-counter">
+        <h3><%= @user.name%></h3>
+        <p>Number of posts: <%= @user.post_counter%></p>
+        </div>
+      </div>
+    </article>
+
+    <article class="users-posts">
+      <% if @posts.empty? %>
+        <h3>Currently there are not posts in the database for this user</h3>
+      <% else %>
+        <% @posts.each do |post| %>
+          <div class="posts-section">
+            <div class="posts-card">
+              
+            <a href='posts/<%=post.id%>'><h4>Post #: <%= post.id %> - <%= post.title %></a></h4>
+        <p class="post-text" maxlength= 150><%= post.text %></p>
+        <p class="post-comments-likes">
+        Comments: <%= post.comments_counter %> - Likes: <%= post.likes_counter %>
+      </p>
```

Comment:
- [ OPTIONAL ] I think it would be better if you use partial files on a specific `Post` for best practices. I suggest you refer this [article](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials) to get a short reminder and examples